Database/test/specs/database.test.js

```diff
@@ -0,0 +1,52 @@
+const axios = require("axios");
+const { Client } = require("pg");
+const pet = require("../utils/pet.json");
+const client = new Client({ 
+  host: "localhost",
+  user: "postgres",
+  port: 5432,
+  password: "12345",
+  database: "postgres" 
+});
+
+describe("API to database collection", function () {
+  before(function () {
+    client.connect();
+    client.query(
+      "INSERT INTO pets (id, name) VALUES (1, 'Bentley'), (2, 'Mari'), (3, 'Tank'), (4, 'Marley'), (5, 'Milo'), (6, 'Gunner'), (7, 'Louie'), (8, 'Thor'), (9, 'Cooper'), (10, 'Leo')"
+    );
+  });
+
+  after(function () {
+    client.query("TRUNCATE TABLE pets");
+  });
```

Comment:
Try to create separate class for this methods and name it hooks

Reply:
Couldn't implement it